lib/DDG/Spice/DebVersion.pm

```diff
@@ -1,9 +1,18 @@
-# Get the version of a package in various debian repositories
 package DDG::Spice::DebVersion;
 
```

Comment:
If you could put line 1 under the package declaration with the ABSTRACT declaration comment, it should be ok...